packages/stellar-axelar-std/src/tests/test.rs

```diff
@@ -26,6 +26,31 @@ mod operatable {
         pub fn __constructor(env: &Env, operator: Address) {
             stellar_axelar_std::interfaces::set_operator(env, &operator);
         }
+
+        #[only_operator]
+        pub fn operator_function(env: &Env) {}
+    }
+
+    #[test]
+    fn operator_function_succeeds_with_correct_operator() {
+        let env = Env::default();
+        let operator = Address::generate(&env);
+        let contract_id = env.register(Contract, (operator.clone(),));
+        let client = ContractClient::new(&env, &contract_id);
+
+        assert_auth!(operator, client.operator_function());
+    }
+
+    #[test]
+    #[should_panic(expected = "InvalidAction")]
+    fn operator_function_fails_with_incorrect_operator() {
+        let env = Env::default();
+        let operator = Address::generate(&env);
+        let non_operator = Address::generate(&env);
+        let contract_id = env.register(Contract, (operator,));
+        let client = ContractClient::new(&env, &contract_id);
+
+        assert_auth!(non_operator, client.operator_function());
```

Comment:
We have a macro to check auth panic with `InvalidAction`, use `assert_auth_err`
```suggestion
        assert_auth_err!(non_operator, client.operator_function());
```